django/contrib/admin/helpers.py

```diff
@@ -36,7 +36,9 @@ class ActionForm(forms.Form):
     )
 
 
-checkbox = forms.CheckboxInput({"class": "action-select"}, lambda value: False)
+checkbox = forms.CheckboxInput(
+    {"class": "action-select", "autocomplete": "off"}, lambda value: False
```

Comment:
I made use of https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autocomplete